error_test.go

```diff
@@ -6,6 +6,58 @@ import (
 	"testing"
 )
 
+func TestErrorSourceSyntaxError(t *testing.T) {
+	tests := []struct {
+		name     string
+		src      string
+		expected string
+	}{
+		{
+			name:     "Inline",
+			src:      "local p = person.new('Steeve');lokal t = 'fail'",
+			expected: "Line 1: 't':   parse error",
```

Comment:
can we trim the extra space between the error and the description?

Reply:
That's the error as it comes from gopher-lua, so I didn't want to tamper with it, but sure, let me try

Reply:
There, nailed it